Puppetfile

```diff
@@ -19,11 +19,7 @@ mod 'puppet-prometheus',                   '6.2.0'
 mod 'puppetlabs-apache',                   '3.1.0'
 mod 'puppetlabs-apt',                      '6.3.0'
 mod 'puppetlabs-concat',                   '4.2.1'
-# Make sure to update alexharvey/firewall_multi version
```

Comment:
this comment should have been kept @ja5087

Reply:
oh yikes

Reply:
nw